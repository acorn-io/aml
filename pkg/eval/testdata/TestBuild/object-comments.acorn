// Block 1 line 1
// Block 1 line 2

// Block 2 line 1

// Block 3 before Object
{}

// After object
