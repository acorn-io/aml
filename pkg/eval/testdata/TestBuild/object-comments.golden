&eval.File{Body: &eval.Struct{
	Position: eval.Position{
		Filename: "object-comments.acorn",
		Offset:   81,
		Line:     7,
		Column:   1,
	},
	Comments: eval.Comments{Comments: [][]string{
		{
			"Block 1 line 1",
			"Block 1 line 2",
		},
		{"Block 2 line 1"},
	}},
	Fields: []eval.Field{&eval.Embedded{
		Pos: eval.Position{
			Filename: "object-comments.acorn",
			Offset:   81,
			Line:     7,
			Column:   1,
		},
		Comments: eval.Comments{Comments: [][]string{
			{"Block 3 before Object"},
			{"After object"},
		}},
		Expression: &eval.Struct{Position: eval.Position{
			Filename: "object-comments.acorn",
			Offset:   81,
			Line:     7,
			Column:   1,
		}},
	}},
}}
